doc/how_to_guides/how_to_setup_docker_containers_in_ubuntu.rst

```diff
@@ -1,68 +1,44 @@
-It includes a script that will get you up and running in MoveIt quickly!
+It includes a docker-compose config file that will get you up and running in MoveIt quickly!
 This guide is intended for people who would like to have a separate environment for working with MoveIt up and running quickly \
 without having to do much configuring. In this guide, we will be setting up a ROS2 Rolling environment.
 
 Learning Objectives
 -------------------
 
-- How to setup a Docker environment using the provided script
+- How to setup a Docker environment using the provided docker-compose config
 
 Requirements
 ------------
 
 - Ubuntu 20.04 or 22.04
 - `Docker Installation for Ubuntu <https://docs.docker.com/engine/install/ubuntu/>`_
 - `Nvidia drivers for Docker <https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#setting-up-nvidia-container-toolkit>`_
+- `docker-compose (if not already installed via Docker Desktop) <https://docs.docker.com/compose/install/>`_
 
 Steps
 -----
-1. Install Docker (a link is available in the Requirements section) and be sure to follow the `Linux Post Install <https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user>`_ instructions. If you do not complete these additional steps you will need to preface all ``docker`` commands with ``sudo``.
+1. Install Docker and docker-compose (links are available in the Requirements section) and be sure to follow the `Linux Post Install <https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user>`_ instructions. If you do not complete these additional steps you will need to preface all ``docker`` commands with ``sudo``.
 
-2. Open a terminal session, download the Docker script, and make it executable.
+2. Open a terminal session, download the config, and make it executable.
 
   .. code-block:: bash
 
-    wget https://raw.githubusercontent.com/abake48/moveit2_tutorials/how-to-docker-ubuntu/_scripts/start-docker.sh -O ~/.local/bin/start-docker.sh
-    chmod +x ~/.local/bin/start-docker.sh
+    wget https://raw.githubusercontent.com/abake48/moveit2_tutorials/how-to-docker-ubuntu/_scripts/docker-compose.yml
 
-3. Run the script.
+3. Launch the container.
 
-  There are 3 parameters for the script:
-      - ``name_of_the_container`` : this is the name you wish to give the created container. For this guide, we will be naming the container ``moveit2-rolling``.
-      - ``name_of_the_image`` : if you are creating a fresh Docker container, provide the name of the Docker image here. For this guide, we will be using the image ``moveit/moveit2:rolling-source``. Further explanation of this parameter is provided in the ``Further Reading`` section.
-      - ``using_gpu`` : if ``true``, the Docker will be run using Nvidia GPU drivers. By default, this value is true.
+   .. code-block:: bash
 
-  To run the script and use Nvidia GPU drivers
+    ROS_DISTRO=rolling-source docker-compose run gpu
```

Comment:
I had to do this without dash:
```suggestion
    ROS_DISTRO=rolling-source docker compose run gpu
```

Reply:
A bit nitpicky but the name `ROS_DISTRO` is a bit confusing here. I would prefer to either one name: DOCKER_IMAGE or some sorts; or have two different parameters, ROS_DISTRO (Which would be rolling, humble, foxy etc.) and BUILD, which would be either source or release.

Reply:
Of course we should change the documentation according to new parameter

Reply:
Interesting- apparently `docker compose` without the hyphen is a new thing :) will add a note that the hyphen may still be required for older versions of compose (and change the env variable name)